src/bundles/SellerRegistration/components/BusinessDetailsForm/BusinessDetailsForm.js

```diff
@@ -126,16 +127,14 @@ class BusinessDetailsForm extends BaseForm {
                           id="abn"
                           htmlFor="abn"
                           label="ABN"
-                          description={isNumber(supplierCode) ? "You need an Australian Business Number to do business in Australia." :
-                              (<span>You need an Australian Business Number to do business in Australia.&nbsp;
-                              <a href='https://abr.gov.au/For-Business,-Super-funds---Charities/Applying-for-an-ABN/' target="_blank" rel="external noopener noreferrer">Apply for an ABN here.</a>
-                          </span>)}
-                          readOnly={isNumber(supplierCode)}
-                          disabled={isNumber(supplierCode)}
+                          readOnly={ABN ? true : false}
+                          disabled={ABN ? true : false}
+                          validators={{
+                              validABN: val => val === 'N/A' || validABN(val)
+                          }}
```

Comment:
The desired behaviour here is to disable the ABN field when it is present in the application data, which it should be on new signups since this value is gathered on signup and pre-filled in the initial application data.

There will be a time period where sellers who started their initial applications before this PR will not have had the ABN value pre-filled. If they still haven't input a valid value by the time this PR is merged and deployed, then this code should still allow them to do so (as `ABN` should be falsey). If they did input a value before this PR, then the field will be disabled.

In the event they somehow managed to provide an invalid value before this PR, then they may need manual ops intervention to fix it to finish the application.